HelpSource/Guides/standalones.schelp

```diff
@@ -1,212 +1,349 @@
+		^[String.scDir +/+ \"startup.scd\"];
 	}
-
 }
+"
+);
+~overDir.openOS;
+)
+
+(
+// 5. write a basic startupFile:
+~startupCode =
+"// basic example startup file for osx standalone.
+// avoid clash with default SC server
+s.addr.port = 57105;
+
+// boots server and plays a little sound
+s.waitForBoot {
+	{
+		Line.kr(1, 0, 10, doneAction: 2) * SinOsc.ar(1000)
+		* EnvGen.ar(Env.perc(0.01, 0.25), Impulse.kr(2, [0, 0.5]), 0.1)
+
+	}.play;
+};
+";
+~writeText.(~newAppResDir +/+ "startup.scd", ~startupCode);
+(~newAppResDir +/+ "startup.scd").openOS;
+)
+
+// *** time for a second wakeup kiss *** //
+// should run the startup file and play a sound:
+unixCmd("open" + ~pathToNewApp);
+
+
+// 6. For larger projects:
+//    make an example project folder with a loadfile in it,
+//    and a startup file that points to the loadfile.
+(
+~projDirName = ~newAppName ++ "_files";
+~projDir = ~newAppResDir +/+ ~projDirName;
+File.mkdir(~projDir);
+~projDir.openOS;
+
+~loadFileCode =
+"\"*** startup file for % loading.\".postln;
+\"*** My Platform.userAppSupportDir is: \".postln;
+Platform.userAppSupportDir.postcs;
+thisProcess.nowExecutingPath.openOS;
+
+// configure server options here if needed:
+s.options.memSize = 8192 * 16;
+// move server port away from default (57110)
+// to avoid interference/clashes with SuperCollider itself:
+s.addr.port = 57105;
+
+s.waitForBoot {
+	\"*** server booted, plays ping.\".postln;
+	Env.perc.test;
+
+	\"*** loading files next ? \".postln;
+	// put your loading sequence here:
+	// \"loadX.scd\".loadRelative;
+	// s.sync; // e.g. when loading buffers
+ 	// 1.wait;
+ 	// \"loadY.scd\".loadRelative;
+};"
+.format(~newAppName);
+
+~loadfilepath = ~projDir +/+ "00_loadMe.scd";
+~writeText.value(~loadfilepath, ~loadFileCode);
+~loadfilepath.openOS;
+
+
+// write a startupFile that points to the load file:
+~startupCode =
+"// example startup file for osx standalone.
+// running loadfile in this project folder:
+%.loadRelative;
+".format((~projDirName +/+ ~loadfilepath.basename).cs);
+
+~writeText.(~newAppResDir +/+ "startup.scd", ~startupCode);
+(~newAppResDir +/+ "startup.scd").openOS;
+)
+
+// *** time for the third wakeup kiss *** //
+unixCmd("open" + ~pathToNewApp);
+
+// this should open the new app, and its loadfile!
+
+
+// 7. create an internal extensions folder inside the new app,
+// and copy all the currently active includePaths there:
+(
+~newAppQuarksDir = ~newAppResDir +/+ "InternalExtensions";
+
+// copy all folders in my current quarks includePaths
+// to Resources/SCClassLibrary to freeze them:
+File.mkdir(~newAppQuarksDir);
+
+~pathsToCopy = LanguageConfig.includePaths.reject { |path|
+	["SCClassLibrary", "HelpSource"].includesEqual(path.basename)
+};
+
+~pathsToCopy.collect(_.basename).sort.printcsAll;
+
+if (~newAppQuarksDir.pathMatch.notEmpty) {
+	"/*** copying Quarks : ***/".postln;
+	~pathsToCopy.do { |path|
+		unixCmd(
+			"cd" + quote(~newAppQuarksDir) ++ ";"
+			" cp -ir" + quote(path).postln + ".");
```

Comment:
weird - I dont get a parse error here (3.9 dev). SO, I added the ++ as suggested just to make sure, 
and it still runs fine.